packages/connection/src/Manager.php

```diff
@@ -43,6 +193,313 @@ public function initialize( $methods ) {
 		$methods;
 	}
 
+	/**
+	 * Since a lot of hosts use a hammer approach to "protecting" WordPress sites,
+	 * and just blanket block all requests to /xmlrpc.php, or apply other overly-sensitive
+	 * security/firewall policies, we provide our own alternate XML RPC API endpoint
+	 * which is accessible via a different URI. Most of the below is copied directly
+	 * from /xmlrpc.php so that we're replicating it as closely as possible.
+	 */
+	public function alternate_xmlrpc() {
+		// phpcs:disable PHPCompatibility.Variables.RemovedPredefinedGlobalVariables.http_raw_post_dataDeprecatedRemoved
+		// phpcs:disable WordPress.WP.GlobalVariablesOverride.Prohibited
+		global $HTTP_RAW_POST_DATA;
+
+		// Some browser-embedded clients send cookies. We don't want them.
+		$_COOKIE = array();
+
+		// A bug in PHP < 5.2.2 makes $HTTP_RAW_POST_DATA not set by default,
```

Comment:
Given our current PHP min version support, I wonder if we could finally get rid of that hack.

Reply:
Removed 👍